grate/Migration/AnsiSqlDatabase.cs

```diff
@@ -256,7 +256,7 @@ private async Task<bool> RunSchemaExists()
     {
         string sql = $"SELECT s.schema_name FROM information_schema.schemata s WHERE s.schema_name = '{SchemaName}'";
         var res = await ExecuteScalarAsync<string>(ActiveConnection, sql);
-        return res == SchemaName;
+        return res != null; // #230: If the server found a record that's good enough for us
```

Comment:
Will it be `null`, or can it be `DBNull`?

Reply:
AFAIK handling the DBNull returned from the sql client is Dapper's job, and it converts to either `null` or `default()` depending on the type it's trying to map to.

If you have any info to the contrary I'd be very keen to know about it!